lib/msal-common/src/request/BaseAuthRequest.ts

```diff
@@ -48,4 +49,6 @@ export type BaseAuthRequest = {
     tokenQueryParameters?: StringDict;
     storeInCache?: StoreInCache;
     scenarioId?: string;
+    popKid?: string;
+    signPopToken?: boolean;
```

Comment:
Can we use the presence of popKid to imply signPopToken=false instead of exposing this as a flag? What's the behavior if someone passes both popKid and signPopToken=true?

Reply:
Addressed this by adding an error that will be thrown if both popKid and signPopToken=true